reference_frame: "gps_origin"

control_action_interval: 0.01 # [s]
uvdar_msg_interval: 0.05 # [s]

# offset from the leader position to be maintained
desired_offset:
  x: -4.0 # [m]
  y: 4.0 # [m]
  z: 0.0 # [m]

# rotation offset around the world vertical axis
heading_offset: 0.43 # [rad]

# custom parameters for pose
safe_distance: 4.00                 # [m]
observation_distance: 2.00          # [m]
velocity_estimator_threshold: 0.4   # [m]
camera_heading_offset: 1.15         # [rad]
use_perception_aware: true          # [bool]
use_trajectory_reference: true      # [bool]
use_speed_tracker: false            # [bool]
path_window_size: 15                # [int]
estimator_weight: 0.00              # [0.0 - 1.0]
traj_prediction_horizon: 0.5        # [s]
replanning_interval_s: 1.0          # [s]

# Noise parameters of the Kalman velocity estimator

# Process noise
Q: [15, 0, 0, 0, 0, 0,
    0, 15, 0, 0, 0, 0,
    0, 0, 15, 0, 0, 0,
    0, 0, 0, 5, 0, 0,
    0, 0, 0, 0, 5, 0,
    0, 0, 0, 0, 0, 5]

# # Measurement noise
R: [75, 0, 0,
    0, 75, 0,
    0, 0, 75]

# R: [50, 0, 0, 0, 0, 0,
#     0, 50, 0, 0, 0, 0,
#     0, 0, 50, 0, 0, 0,
#     0, 0, 0, 5, 0, 0,
#     0, 0, 0, 0, 5, 0,
#     0, 0, 0, 0, 0, 5]